### PR TITLE
Raiden private node & console fixes

### PR DESCRIPTION
**Pull request by rakanalh**
Fixes in this PR allowed me to be able to run Raiden on a local geth node as well as running console which have tools that i used for deploying a token ... etc

### CODE REVIEW

File: raiden/ui/console.py [hunk 1]
```diff
@@ -234,51 +232,6 @@ def _run(self):  # pylint: disable=method-hidden
...
-        err = io.StringIO()
-        sys.stderr = err
-
-        def lasterr(n=1):
-            """ Print the last `n` entries of stderr to stdout. """
-            for line in (err.getvalue().strip().split('\n') or [])[-n:]:
-                print(line)
-
-        self.console_locals['lasterr'] = lasterr
-
```
Comment (ulope) [inline raiden/ui/console.py]: Removing this will cause the console to be spammed by log lines and thereby be almost unusable. Is that intentional?

**Comment by rakanalh** [comment]
ahh just realized this is "borsed", i'll revert the log suppression in console in a different PR

**Comment by palango** [comment]
bors retry

**Pull request merged**